src/applications/edu-benefits/feedback-tool/helpers.js

```diff
@@ -22,6 +22,27 @@ const searchToolSchoolAddressFields = get(
   fullSchema
 );
 
+// The flags to add to formData that will indicate if a page is prefilled or not
+// These flags will be used for each form page's call to
+// conditionalPrefillMessage()
+export const PREFILL_FLAGS = {
+  // if formData.fullName is set:
+  APPLICANT_INFORMATION: 'view:prefilledApplicantInformation',
```

Comment:
I think `prefilledApplicantInformation` suggests that this is the prefill data where `applicationInformationPrefilled` suggests boolean.

Reply:
Well now I'm really happy I put these on the `PREFILL_FLAGS` so it's easy to change :)

Reply:
how about applicantInformationPrefilled?